packages/grid/x-data-grid/src/components/cell/GridActionsCellItem.tsx

```diff
@@ -1,54 +1,55 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import { IconButtonProps } from '@mui/material/IconButton';
-import MenuItem, { MenuItemProps } from '@mui/material/MenuItem';
-import ListItemIcon from '@mui/material/ListItemIcon';
```

Comment:
@cherniavskii For the type's usage from Material UI, what do you think about this approach?

Create a generic type, so that this component is design-agnostic. And at the index.ts, fill the generic with the types from Material or Joy UI.

Developer that `import { GridActionsCellItem } from '@mui-x/data-grid'` will still get the same types from Material UI.

Developer that `import { GridActionsCellItem } from '@mui-x/data-grid/joy'` will get types from Joy UI.